state-chain/chains/src/dot.rs

```diff
@@ -27,7 +27,7 @@ use scale_info::TypeInfo;
 use sp_core::{sr25519, ConstBool, H256};
 
 #[cfg_attr(feature = "std", derive(Hash))]
-#[derive(Debug, Encode, Decode, TypeInfo, Eq, PartialEq, Clone)]
+#[derive(Debug, Encode, Decode, TypeInfo, Eq, PartialEq, Clone, Serialize, Deserialize)]
```

Comment:
Are these still required? Or could we revert all dot and evm related to their previous state?